### ISSUE TITLE
Can't connect to Dbus

### ISSUE THREAD

**Issue by hotice** [Douane/Douane#11]
Douane can't connect to D-Bus. Here's what I get when running the configurator via terminal:

⌁ [andrei:~] $ /usr/bin/configurator

*\* (configurator:23540): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-9GzcTxZqQO: Connection refused
/usr/lib/python3/dist-packages/gi/overrides/GLib.py:53: PyGIDeprecationWarning: threads_init longer needs to be called. See: https://bugzilla.gnome.org/show_bug.cgi?id=686914
  PyGIDeprecationWarning)
[TweeterBox] ERROR: HTTP Error 410: Gone
ln: failed to create symbolic link ‘/lib/modules/3.10.0-2-generic/updates/dkms/douane.ko’: File exists
ERROR: fstatat(5, douane.ko): No such file or directory
FATAL: Module douane not found.
- Starting the douaned daemon                                                                                                                                                          [ OK ] 
  [TweeterBox] ERROR: HTTP Error 410: Gone
  [TweeterBox] ERROR: HTTP Error 410: Gone
  [TweeterBox] ERROR: HTTP Error 410: Gone

This happens each time I start the configurator.
